src/components/ProjectIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import ShowChartIcon from '@mui/icons-material/ShowChart';
+import { Box } from '@mui/material';
+
+export const ProjectIcon = () => {
+  return (
+    <Box
+      sx={{
+        bgcolor: 'project.main',
+        borderRadius: '0.4rem',
+        height: '1.6rem',
+        width: '1.6rem',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}>
+      <ShowChartIcon />
+    </Box>
```

Comment:
Flex med bare ett element er oftest overflødig. Og her ser det ut som at det egentlig er selve ikonet vi ønsker å style, og ikke en parent container? Sjekka fort og ser ut som at dette gjør det samme, men meget mulig jeg misser noe!
```suggestion
    <ShowChartIcon
      sx={{
        bgcolor: 'project.main',
        borderRadius: '0.4rem',
      }}
    />
```